docs/docsite/rst/user_guide/become.rst

```diff
@@ -130,28 +133,57 @@ that only allow reading by the user and root, or only allow reading by the unpri
 user being switched to.
 
 However, when both the connection user and the ``become_user`` are unprivileged,
-the module file is written as the user that Ansible connects as, but the file needs to
-be readable by the user Ansible is set to ``become``. In this case, Ansible makes
-the module file world-readable for the duration of the Ansible module execution.
-Once the module is done executing, Ansible deletes the temporary file.
+the module file is written as the user that Ansible connects as (the
+``remote_user``), but the file needs to be readable by the user Ansible is set
+to ``become``. Ansible solves this problem in the following way:
+
+First, if :command:`setfacl` is installed and available in the remote ``PATH``,
```

Comment:
i would note 'this depends on the platform, but for POSIX systems this is how it works' or similar phrasing